dask/tests/test_dot.py

```diff
@@ -46,3 +47,9 @@ def test_dot_graph():
 def test_aliases():
     g = to_networkx({'x': 1, 'y': 'x'})
     assert 'y' in g.edge['x']
+
+
+def test_np_graph():
+    g = to_networkx({'x': np.empty([1, 2])})
+    assert g.node['x']['label'] == 'x=[[ 0.  1.]]' 
```

Comment:
Why the `[ 0. 1.]`?  Is this what happened to be in memory?  Perhaps change x to `np.array([1, 2])`?